### ISSUE TITLE
Scan our system for vulnerabilities

### ISSUE THREAD

**Issue by JulioDeLeon** [JulioDeLeon/CSE-545-Team-13-Central#9]
Description: we need to be able to scan our system for vulnerabilities.
Requirements:
- be able to list the type of vulnerabilities found as output 
- if possible, use an off-the-shelf solution to scan the system.

Nice to have:
- Have the scan automated

**Comment by JulioDeLeon** [comment]
depends on: https://github.com/JulioDeLeon/CSE-545-Team-13-Central/issues/5

**Comment by Pactopolis** [comment]
Since @JulioDeLeon has already started pre-req work on this, putting back on TODO
